.eslintrc.json

```diff
@@ -0,0 +1,50 @@
+{
+    "extends": "eslint:recommended",
+    "env": {
+        "es2019": true,
+        "node": true
+    },
+    "parserOptions": {
+        "ecmaVersion": 2019,
+        "sourceType": "script"
+    },
+    "rules": {
+        "semi": ["error", "always"],
+        "quotes": ["error", "single"],
+        "eqeqeq": ["error", "always", { "null": "ignore" }],
+        "guard-for-in": ["error"],
+        "no-extend-native": ["error"],
+        "wrap-iife": ["error"],
+        "indent": ["error", 4],
+        "no-use-before-define": ["error", { "functions": false }],
+        "new-cap": ["error"],
+        "no-caller": ["error"],
+        "strict": ["error"],
+        "no-trailing-spaces": ["error"],
+        "max-params": ["error", 10],
+        "max-depth": ["error", 5],
+        "max-statements": ["error", 25],
+        "max-len": ["error", 110],
+        "no-var": ["error"],
+        // Allow unused vars if prefixed with "_"
+        "no-unused-vars": ["error", { "argsIgnorePattern": "^_" }],
```

Comment:
The previous rules just turned off this check. I modified things slightly here to allow prefixing unused arguments with `_`. It’s useful so you can show how they are present in a callback even if not using them, but get the check in other cases where you accidentally left an argument or variable around that you didn’t mean to.